packages/utils/src/post-process.ts

```diff
@@ -260,6 +290,22 @@ export function postProcessBundle(
     MemberExpression(path) {
       const { node } = path;
 
+      // Replace process.env with raw string.
+      if (
+        dotenv !== undefined &&
+        path.get('object').matchesPattern('process.env')
+      ) {
+        const key = toComputedKey(node);
+        if (
+          isStringLiteral(key) &&
+          !isLeftSideOfAssignment(path) &&
+          key.value in dotenv
+        ) {
+          path.replaceWith(valueToNode(dotenv[key.value]));
+          (visitor.StringLiteral as any)(path);
```

Comment:
What does this do?

Reply:
Calls the post processing on the new string.

So if .env file is
```
SNAP_ASD=import()
```

this will properly replace the newly created `"import()"` literal with `"import\()"`

Reply:
Looks like this works without explicitly calling the visitor too, so you can remove this line. AFAIK it always revisits a node, unless you call `path.skip()`.

```suggestion
```